HIP/hip-227.md

```diff
@@ -71,7 +74,6 @@ Optional filters
 - `limit`
 - `order`
 - `timestamp`
```

Comment:
We should document what values are supported like the comparison operators or if it's hex-encoded and `0x` is optional, etc.

Reply:
Added